packages/fast-components-react-msft/src/background/background.tsx

```diff
@@ -48,7 +48,8 @@ export default class Background extends Foundation<
                 ? background
                 : has(designSystem.neutralPalette, background)
                     ? get(designSystem.neutralPalette, background)
-                    : DesignSystemDefaults.neutralPalette[Background.defaultProps.value];
+                    : DesignSystemDefaults.neutralPalette[background] ||
```

Comment:
Bad merge?

Reply:
uhhhh not really - needed it to fix this issue. It will get resolved when we rebase.